site/_docs/history.md

```diff
@@ -30,39 +30,127 @@ Downloads are available on the
 
 <!--
 
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.38.0">1.38.0</a> / YYYY-MM-DD
-{: #v1-38-0}
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.39.0">1.39.0</a> / YYYY-MM-DD
+{: #v1-39-0}
 
-This release comes x months after [1.37.0](#v1-37-0),
+This release comes x months after [1.38.0](#v1-38-0),
 contains contributions from x contributors, and resolves x issues.
+
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 23;
+Guava versions 21.0 to 33.3.0-jre;
+other software versions as specified in gradle.properties.
+
 #### Breaking Changes
-{: #breaking-1-38-0}
+{: #breaking-1-39-0}
+
+None.
+
+#### New features
+{: #new-features-1-39-0}
+
+#### Dependency version upgrade
+{: #dependency-1-39-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-39-0}
+
+#### Build and test suite
+{: #build-1-39-0}
 
-In previous versions of Calcite the casts to DECIMAL types were
-treated as no-ops [CALCITE-6322].  Fixing this bug causes all
-calculations that use DECIMAL values to produce slightly different
-results.
+#### Web site and documentation
+{: #site-1-39-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.38.0">1.38.0</a> / 2024-10-09
+{: #v1-38-0}
+
+This release comes 6 months after [1.37.0](#v1-37-0),
+contains contributions from 40 contributors, and resolves 162 issues.
+Highlights include the
+[`AS MEASURE`](https://issues.apache.org/jira/browse/CALCITE-4496)
+clause to define measures and use them in
+[simple queries](https://issues.apache.org/jira/browse/CALCITE-6519),
+[`ASOF` join](https://issues.apache.org/jira/browse/CALCITE-6372),
+the
+[`EXCLUDE`](https://issues.apache.org/jira/browse/CALCITE-5855)
+clause in window aggregates, Postgres-compatible implementations of the
+[`TO_DATE`, `TO_TIMESTAMP`](https://issues.apache.org/jira/browse/CALCITE-6449)
+and
+[`TO_CHAR`](https://issues.apache.org/jira/browse/CALCITE-6358)
+functions, and the extension of the type system to allow
+[types with negative scale](https://issues.apache.org/jira/browse/CALCITE-6560).
+
+Contributors to this release:
+Aleksey Plekhanov,
+Alessandro Solimando,
+Barry Kelly,
+Bowen Yang,
+Cancai Cai,
+Clay Johnson,
+Cyril de Catheu,
+Dawid Wysakowicz,
+Evgeniy Stanilovsky,
+Fan Luo,
+Gian Merlino,
+Ian Bertolacci,
+Itiel Sadeh,
+James Duong,
+Jiabao Sun,
+Jiajun Xie,
+jianhong.hu,
+Jie Cheng,
+Julian Hyde (release manager),
+Konstantin Orlov,
+Krisztian Kasa,
+Michael Mior,
+Mihai Budiu,
+Mou Wu,
+Niels Pardon,
+Nitish Kumar,
+Norman Jordan,
+Pranava B,
+Rodrigo Rueda,
+Ruben Quesada Lopez,
+Sergey Nuyanzin,
+Stamatis Zampetakis,
+suibianwanwan,
+Tim Grein,
+TJ Banghart,
+Wegdan Ghazi,
+xiaohei,
+Xiong Duan,
+YiwenWu,
```

Comment:
It seems that the name is Yiwen Wu and I found the following entries in the git log:
* YiwenWu <yiwenwu@tencent.com>
* YiwenWu <yiwenwu1006@gmail.com>